src/search/result.tsx

```diff
@@ -18,20 +17,20 @@ export const Result: React.SFC<ResultProps> = ({ company, onClickResult }) => {
   const click = () => onClickResult(company)
 
   return (
-    <li onClick={click} className="search-container__result">
+    <Li onClick={click}>
       {company.name} ({company.symbol})
-    </li>
+    </Li>
   )
 }
 
 class SearchResults extends React.PureComponent<SearchResultProps> {
   public render() {
     return (
-      <ul className="search-container__results-list">
+      <UnlistedList>
```

Comment:
Minor thing, maybe either rename `Li` to `List` or rename `UnlistedList` to `Ul` to keep naming consistent.